src/ImageBuilder/ImageBuilder.Autorest/custom/New-AzImageBuilderTemplateSourceObject/New-AzImageBuilderTemplateSourceObject_ManagedImageSource.ps1

```diff
@@ -21,36 +21,27 @@ Create an in-memory object for ImageTemplateSource.
 Create an in-memory object for ImageTemplateSource.
 
 .Outputs
-Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.ImageTemplateManagedImageSource
+Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.ImageTemplateManagedImageSource
 .Link
 https://learn.microsoft.com/powershell/module/az.ImageBuilder/new-azimagebuildertemplatesourceobject
 #>
 function New-AzImageBuilderTemplateSourceObject_ManagedImageSource {
-    [OutputType('Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.ImageTemplateManagedImageSource')]
```

Comment:
For cmdlet like this one(polymorphism), there are several options.

1. Option 1: create cmdlet per each child (autogen)
2. Option 2: keep it as it is (custom). Using a parameter set for each child